### PR TITLE
Added toggle styling and tooltip to describe usage

### LINKED ISSUE #208: Widget drawer toggle styling.
**Is your feature request related to a problem? Please describe.**
Styling must be added to the widget drawer toggle in order to make it conform with the rest of the component.

**Describe the solution you'd like**
Add CSS to the module and add labels for left and right in order to make it clear how the toggle is used.

**Describe alternatives you've considered**
Not having any styling which will negatively affect the aesthetics and usability.

### PR DESCRIPTION
**Pull request by a-158w**
Closes #208

## Proposed Changes

- Toggle switch moved into widget drawer so colour is consistent.
- Tooltip added to explain how to use the toggle to add widgets on the left and right sides of the journal.

image
image

### CODE REVIEW

File: telos-frontend/src/components/Widget.module.css [hunk 1]
```diff
@@ -42,4 +42,4 @@
   line-height: var(--bigTextSize);
   font-weight: 600;
   color: var(--iconColour);
-}
+}
```
Comment (milkyway404) [inline telos-frontend/src/components/Widget.module.css]: missing a newline here

**Review by milkyway404 — CHANGES_REQUESTED** [review]
Thanks for the changes, great idea to add a tooltip! Could you please add images to the PR?

Also, please use `npm ci` instead of `npm install` so you don't get these massive differences in package-lock.json and push again.

**Review by JChamberlain28 — COMMENTED** [review]
Changes are reasonable. However, please provide a screenshot of the tooltip and the new toggle location.

File: telos-frontend/src/components/Widget.module.css (hunk 1 above)
Comment (a-158w) [reply telos-frontend/src/components/Widget.module.css]: Fixed

**Review by milkyway404 — APPROVED** [review]
Happy to approve now that images and newline added. I was concerned about the changes to package-lock.json and recommend using `npm ci` next time to avoid these concerns, but I have cloned and run the repository and it works fine.

**Review by JChamberlain28 — APPROVED** [review]
Suggested changes fullfed. Nice work!

**Pull request merged**